### ISSUE TITLE
Question: how to activate explicitly multiple linters for given file types?

### ISSUE THREAD

**Issue by nvtkaszpir** [SublimeLinter/SublimeLinter#1177]
Let say I have ansible files, which are plain .yml files.
How can I add ansible-lint and yaml-lint to such files explicitly?

From what I see the only plugin that activates itself is 'annotations'...

**Comment by kaste** [comment]
ansible-lint probably just works if you have the ansible syntax installed.

For yaml-lint you need to set the linter setting 'selector' probably to "source.ansible".

**Comment by nvtkaszpir** [comment]
Please elaborate in plain english? :)

Right now I just cloned sublimelinter-contrib-yamllint and added 'ansible' to `syntax = ()` set. Works.
But was wondering if there is an option to do it without altering plugin source code...

**Comment by braver** [comment]
Linters take a "selector" setting. It's very new and a bit undocumented, but if set it takes preference over the "syntax" property. 

So try something like this in your settings and tweak it till it works for your ansible file.

```json
"linters": {
    "eslint": {
        "selector": "source.js"
    }
}
```

To add multiple syntaxes, use a comma (e.g. "source.yaml, source.ansible" probably in this case). To find out what selector to use for ansible, use the Show Scope Name feature in an ansible file and take the first string in the popup you get.

**Comment by nvtkaszpir** [comment]
Thanks for the explanation.
This works perfectly.
